config/main.py

```diff
@@ -474,9 +478,11 @@ def add_portchannel_member(ctx, portchannel_name, port_name):
 def del_portchannel_member(ctx, portchannel_name, port_name):
     """Remove member from portchannel"""
     db = ctx.obj['db']
-    db.set_entry('PORTCHANNEL_MEMBER', (portchannel_name, port_name), None)
-    db.set_entry('PORTCHANNEL_MEMBER', portchannel_name + '|' + port_name, None)
-
+    if len(db.get_entry('PORTCHANNEL_MEMBER', portchannel_name + '|' + port_namee)) != 0:
```

Comment:
port_namee ?

Reply:
Thank you. I have addressed  it and updated the pull request.

Reply:
@madhupalu, i'll revert this change. Can you create a new PR that fixes this?